python/vineyard/contrib/ml/tests/test_dali.py

```diff
@@ -32,6 +32,10 @@ def vineyard_for_dali():
         yield
 
 
+def test_dali_tensor_with_rpc_client(vineyard_rpc_client):
+    test_dali_tensor(vineyard_rpc_client)
+
+
```

Comment:
Use `pytest.mark.parametrize` (https://docs.pytest.org/en/7.1.x/how-to/parametrize.html) to avoid these copies, e.g.,

```
@pytest.mark.parametrize("vineyard_client", [vineyard_client, vineyard_rpc_client])
def test....
```